pkg/controller/upload-controller_test.go

```diff
@@ -43,6 +43,14 @@ const (
 	cloneRequestAnnotation  = "k8s.io/CloneRequest"
 )
 
+var (
+	yyytestUploadServerCASecret     *corev1.Secret
```

Comment:
Can you fix the variable names here to not include yyy/zzz/etc?

Reply:
DONE DEAL